src/bin/conserve.rs

```diff
@@ -87,6 +87,18 @@ fn make_clap<'a, 'b>() -> clap::App<'a, 'b> {
             .help("Exclude files that match the provided glob pattern")
     };
 
+    fn include_only_arg<'a, 'b>() -> Arg<'a, 'b> {
```

Comment:
I changed the argument parsing to use `structopt`, so this part will need to be rewritten, but it will be simpler afterwards.